pkg/templating/manifest_generator.go

```diff
@@ -300,8 +306,10 @@ func (i Instance) GenerateAuthenticationAttributes(an AuthenticationMethod) stri
 		attributes = i.generateApiKeyAttributes()
 	case ApplicationID:
 		attributes = i.generateApplicationIdAttributes()
+	case OpenIdConnect:
+		attributes = i.generateOIDCAttributes()
 	case Hybrid:
-		attributes = i.generateApiKeyAttributes() + "\n      " + i.generateApplicationIdAttributes()
+		attributes = i.generateApiKeyAttributes() + "\n      " + i.generateApplicationIdAttributes() + "\n        " + i.generateJWTClaim()
```

Comment:
hm, idea for a new PR: pass in the list of auths you want to support in a "hybrid" template with a flag, ie. for using app_id and OIDC but not apikey, `--hybrid-authn-list=appid,oidc` (or just reuse the current one accepting a list, though perhaps best to use complete strings in CSV format rather than numbers)

Reply:
Yes, sounds like a good idea. Would prefer to keep out of this PR for now though if that’s ok.